app/src/main/java/co/nimblehq/compose/crypto/ui/common/price/PriceChange.kt

```diff
@@ -38,7 +38,7 @@ fun PriceChange(
             } else {
                 painterResource(id = R.drawable.ic_fire_opal_arrow_down)
             },
-            tint = if (isPositiveNumber) Color.GuppieGreen else Color.FireOpal,
+            tint = if (isPositiveNumber) GuppieGreen else FireOpal,
```

Comment:
shouldn't we pass the AppTheme here so you can access its color attributes sth like: `AppTheme.colors.textPricePositive` and `AppTheme.colors.textPriceNegative`?

IMO if we decide to manage colors with Theme, we should never use a color directly 💡 if we have only 1 theme, or 2 themes but using this same color, it's still better to define them under Theme level instead of calling the color directly like this.

Reply:
@sleepylee yes, that's also a valid for refactoring as well 👍